_integration-schemas/3plcentral/inventory.md

```diff
@@ -0,0 +1,262 @@
+---
+tap: "3plcentral"
+version: "1.0"
+
+name: "inventory"
+doc-link: "http://api.3plcentral.com/rels/"
+singer-schema: "https://github.com/singer-io/tap-3plcentral/blob/master/tap_3plcentral/schemas/inventory.json"
+description: "This table contains information about inventory."
+
+replication-method: "Full Table"
+
+api-method:
+    name: "3PL Central REL documentaion"
+    doc-link: "http://api.3plcentral.com/rels/inventory/inventory"
+
+attributes:
+  - name: "receive_item_id"
+    type: "integer"
+    primary-key: true
+    description: ""
+    foreign-key-id: "receive-item-id"
+
+  - name: "available_qty"
+    type: "number"
+    description: ""
+
+  - name: "cost"
+    type: "number"
+    description: ""
+
+  - name: "customer_identifier"
+    type: "object"
+    description: ""
```

Comment:
Another one you can safely guess at: `Details about the customer associated with the inventory record.`

If you can safely guess, go for it. Especially if there's a foreign key (see below) involved.